dd-sdk-android-support-fragment/src/test/kotlin/com/datadog/android/support/fragment/internal/RumMonitorBasedTest.kt

```diff
@@ -22,7 +22,7 @@ import org.mockito.quality.Strictness
     ExtendWith(ForgeExtension::class)
 )
 @MockitoSettings(strictness = Strictness.LENIENT)
-internal abstract class LifecycleCallbacksTest {
+internal abstract class RumMonitorBasedTest {
```

Comment:
I don't understand this name choice ? Why can't you keep `LifecycleCallbacksTest` or `AbstractLifecycleCallbacksTest` ?

Reply:
case is not only the LifecycleCallbacks that inherit from that.